### PR TITLE
Add skeleton for `test_workload_with_fsgroup` test

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#2964

### CODE REVIEW

File: manager/integration/tests/test_basic.py [hunk 1]
```diff
@@ -3663,3 +3663,23 @@ def test_expand_pvc_with_size_round_up(client, core_api, volume_name):  # NOQA
...
+            runAsUser: 1000
+            runAsGroup: 1000
+            fsGroup: 1000
+        ```
+       See
+       https://github.com/longhorn/longhorn/issues/2964#issuecomment-910117570
+       for an example.
+    2. Wait for the workload pod to be running
+    3. Exec into the workload pod, cd into the mount point of the volume.
+    4. Verify that the mount point has correct filesystem permission
```
Comment (khushboo-rancher) [inline manager/integration/tests/test_basic.py:3683]: Can you provide the exact command to check the filesystem permission, or you're referring to `ls -l` ?
I was thinking reading and writing check should be enough for the test.

**Review by innobead — APPROVED** [review]
LGTM

@PhanLe1010 Please help answer @khushboo-rancher question. Thanks.

File: manager/integration/tests/test_basic.py (hunk 1 above)
Comment (PhanLe1010) [reply manager/integration/tests/test_basic.py]: Yeah, I added the command. Thank you

**Review by khushboo-rancher — APPROVED** [review]
LGTM

**Review by innobead — APPROVED** [review]

**Comment by khushboo-rancher** [comment]
@PhanLe1010 Can you please help backport this to v1.2.x so that I can backport https://github.com/longhorn/longhorn-tests/pull/710 too?

**Comment by PhanLe1010** [comment]
I pushed the back port https://github.com/longhorn/longhorn-tests/pull/713

**Pull request merged**